### PR TITLE
Day 2 Task 1 (Program to print sum of elements of dynamic list provided to function using *args)

### PR DESCRIPTION
**Pull request by Atharva321**
Task :WAP to create a function which returns the sum of all the element  provided to it as an argument. The argument list will be dynamic.

What this PR does?
1)Added task1,task2,task3 folder
2)Solution of Day 2 Task 1
3)Readme.md file explaining the solution

### CODE REVIEW

**Comment by Atharva321** [comment]
What improvement should be done in program, Readme.md, PR name and PR description format?

**Comment by Atharva321** [comment]
And  How * inserts the  space in words/multi digit numbers: 
```
>>>string  = input("Enter string : ")
Enter string : 12 34 56
>>>print("Printing string  as it is: ", string)
Printing string  as it is:  12 34 56
>>>print("Printing string using *args :", *string)
Printing string using *args : 1 2   3 4   5 6
>>>"12 34 56".split()
['12','34','56']

```

File: day2/solutions/task1/solution.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        sum += int(passed_number)
+    return(sum)
+
+numbers = (map(int,input("Enter the of numbers: ").split()))
+print("Sum of elements : ", calculate_sum(*numbers))
+string  = input("Enter string : ")
+
+print("Printing string  as it is: ", string)
+
+#It is printing string with space after every character
```
Comment (chowmean) [inline day2/solutions/task1/solution.py]: Need not comment the obvious things. Too much commenting is also a problem.

File: day2/solutions/task1/solution.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+provided to it as an arguement. The arguement list will be dynamic.
+----------------------------------------------------------------------------------
+The *args  i.e. *<variable_name/function>. treats every element seperaterly
+only if arguments provided without key assignment
+"""
+def calculate_sum(*args):
+    print("Type of *args : ", type(args))
+    sum = 0
+    print("Given Elements: ", args)
+#All arguments passed to function are converted into tuple by default.
```
Comment (chowmean) [inline day2/solutions/task1/solution.py]: You have already printed the type of args which will show tuples and that is great. I don't think you need to mention it again here.

File: day2/solutions/task1/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+>>>
+>>>print("Printing string  as it is: ", string)
+>>>Printing string  as it is:  12 34 56
+>>>
+>>>print("Printing string using *args :", *string)
+>>>Printing string using *args : 1 2   3 4   5 6
+```
+
+# Resources that helped to understand better:
+https://stackoverflow.com/questions/18753627/difference-bewteen-args-and-args-in-python
```
Comment (chowmean) [inline day2/solutions/task1/README.md]: Can you convert it to links like this 
```
[Difference between args and kwargs](https://stackoverflow.com/questions/18753627/difference-bewteen-args-and-args-in-python)
```
Markdown will create it to link.

**Review by chowmean — COMMENTED** [review]
Please check the commented parts.

File: day2/solutions/task1/README.md (hunk 3 above)
Comment (Atharva321) [reply day2/solutions/task1/README.md]: Yes done the changes

**Comment by Atharva321** [comment]
I done something wrong while converting two commits into one all changes gone

Commands I used:
`git rebase -i f2593443155aa39ae27d030859a3b0480be7aed6`

then their is only 
`pick e80542f `
in git-rebase-todo file
 I changed it to following code 
```
pick f259344 
squash e80542f 
```

After that 

`git push -f  origin Day_2_task_1 `


now, If I run command
`git rebase -i f2593443155aa39ae27d030859a3b0480be7aed6`

this is in git-rebase-todo file
noop

what i done wrong here

**Comment by Atharva321** [comment]
pushed new commit plz check it out

**Pull request merged**